torchinductor/scheduler.py

```diff
@@ -19,15 +19,20 @@
 from .sizevars import SimplifyIndexing
 from .virtualized import V
 
-TemplateKernels = [ir.Convolution]
+template_kernels = [ir.Convolution]
```

Comment:
I think we could set template_kernels = [] to disable TritonTemplateKernel

Reply:
This is defined before the configuration might get changed, so we will need to move it out of global scope for that to work.

Reply:
oh I missed the config. Makes sense